src/commands/cronjob/update.tsx

```diff
@@ -0,0 +1,153 @@
+import { MittwaldAPIV2 } from "@mittwald/api-client";
+
+type UpdateResult = void;
+
+export default class Update extends ExecRenderBaseCommand<
+  typeof Update,
+  UpdateResult
+> {
+  static description = "Update an existing cron job";
+  static args = {
+    "cronjob-id": Args.string({
+      description: "ID of the cron job to be updated.",
+      required: true,
+    }),
+  };
+  static flags = {
+    description: Flags.string({
+      description: "Set cron job description",
+    }),
+    interval: Flags.string({
+      description: "Set cron job execution interval",
+    }),
+    disable: Flags.boolean({
+      description: "Disable cron job automated execution",
+      exclusive: ["enable"],
+    }),
+    enable: Flags.boolean({
+      description: "Enable cron job automated execution",
+      exclusive: ["disable"],
+    }),
+    email: Flags.string({
+      description: "Set target email to send error messages to",
+    }),
+    timeout: Flags.integer({
+      description: "Set timeout in seconds after wich the process is killed",
+    }),
+    url: Flags.string({
+      description: "Set url to use on cron job execution",
+    }),
+    command: Flags.string({
+      description: "Set file and parameters to execute on cron job execution",
+    }),
+    interpreter: Flags.string({
+      description: "Set interpreter to use for execution",
+      options: ["bash", "php"],
+    }),
+    ...processFlags,
+  };
+
+  protected async exec(): Promise<void> {
+    type destinationType =
+      | MittwaldAPIV2.Components.Schemas.CronjobCronjobCommand
+      | MittwaldAPIV2.Components.Schemas.CronjobCronjobUrl;
+
+    type cronjobUpdatePayload = {
+      active: boolean;
+      description: string;
+      destination: destinationType;
+      email: string;
+      timeout: number;
+      interval: string;
+    };
+
+    const process = makeProcessRenderer(this.flags, "Updating cron job");
+
+    const { "cronjob-id": cronjobId } = this.args;
+    const currentCronjob = await this.apiClient.cronjob.getCronjob({
+      cronjobId,
+    });
+    assertSuccess(currentCronjob);
+
+    const {
+      description,
+      interval,
+      email,
+      timeout,
+      url,
+      interpreter,
+      command,
+      disable,
+      enable,
+    } = this.flags;
+
+    let cronjobActive: boolean;
+    if (enable) {
+      cronjobActive = true;
+    } else if (disable) {
+      cronjobActive = false;
+    } else {
+      cronjobActive = currentCronjob.data.active as boolean;
+    }
+
+    let destination: destinationType = currentCronjob.data
+      .destination as destinationType;
+
+    if (url) {
+      destination = {
+        url,
+      };
+    } else if (interpreter) {
+      let destinationInterpreter = interpreter;
+      if (interpreter == "bash") {
+        destinationInterpreter = "/bin/bash";
+      } else if (interpreter == "php") {
+        destinationInterpreter = "/usr/bin/php";
+      }
+
+      if (url) {
+        destination = { url };
```

Comment:
The `if (url) { ... }` appears twice; and in this case, the condition is impossible to reach, because it's in the `else` branch of the first `if (url)` condition.